policyengine_us/parameters/gov/states/nm/tax/income/credits/cdcc/divisor.yaml

```diff
@@ -0,0 +1,13 @@
+description: New Mexico divides the credit for dependent day care for seprate filers by this divisor.
+values:
+  2021-01-01: 2
+metadata:
+  unit: /1
+  label: New Mexico credit for dependent day care separate divisor
+  reference:
+    - title: New Mexico Administrative Code, § 3.3.13.9 - CREDIT FOR EXPENSES FOR DEPENDENT CHILD DAY CARE NECESSARY TO ENABLE GAINFUL EMPLOYMENT TO PREVENT INDIGENCY
+      href: https://www.law.cornell.edu/regulations/new-mexico/3-3-13-9-NMAC 
+    - title: 2022 New Mexico Income Tax Form Intructions
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf 
```

Comment:
Is the `2` based on the $600/$1,200 at the top of Section V? Add more precise references.

Reply:
Added the right sources
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/9ebbc4f2-21f8-453a-b6d8-4845f86d3fd1)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/623851cd-e243-49db-8fe1-e99c885cce57)